components/brave_wallet_ui/components/desktop/views/portfolio/components/nft-details/index.tsx

```diff
@@ -85,61 +86,79 @@ const NFTDetails = (props: Props) => {
-      </DetailColumn>
+      {isLoading &&
+        <LoadingOverlay isLoading={isLoading}>
+          <LoadIcon />
+        </LoadingOverlay>
+      }
+      {nftMetadata &&
+        <>
+          <NftImageWrapper>
+            <NTFImageIframe
+              src={`chrome-untrusted://nft-display?imageUrl=${encodeURIComponent(nftMetadata.imageURL)}&imageWidth=${nftImageDimension}&imageHeight=${nftImageDimension}`}
+              sandbox="allow-scripts"
+            />
+          </NftImageWrapper>
+          <DetailColumn>
+            <TokenName>
+              {selectedAsset.name} {
+                selectedAsset.tokenId
+                  ? '#' + new Amount(selectedAsset.tokenId).toNumber()
+                  : ''
+              }
+            </TokenName>
+            {/* TODO: Add floorFiatPrice & floorCryptoPrice when data is available from backend: https://github.com/brave/brave-browser/issues/22627 */}
+            {/* <TokenFiatValue>{CurrencySymbols[defaultCurrencies.fiat]}{nftMetadata.floorFiatPrice}</TokenFiatValue> */}
+            {/* <TokenCryptoValue>{nftMetadata.floorCryptoPrice} {selectedNetwork.symbol}</TokenCryptoValue> */}
+            <DetailSectionRow>
+              <DetailSectionColumn>
+                <DetailSectionTitle>{getLocale('braveWalletNFTDetailBlockchain')}</DetailSectionTitle>
+                <DetailSectionValue>{nftMetadata.chainName}</DetailSectionValue>
+              </DetailSectionColumn>
+              <DetailSectionColumn>
+                <DetailSectionTitle>{getLocale('braveWalletNFTDetailTokenStandard')}</DetailSectionTitle>
+                <DetailSectionValue>{nftMetadata.tokenType}</DetailSectionValue>
+              </DetailSectionColumn>
+              <DetailSectionColumn>
+                <DetailSectionTitle>{getLocale('braveWalletNFTDetailTokenID')}</DetailSectionTitle>
+                <ProjectDetailIDRow>
+                  <DetailSectionValue>
+                    {
+                      selectedAsset.tokenId
+                        ? '#' + new Amount(selectedAsset.tokenId).toNumber()
+                        : ''
+                    }
+                  </DetailSectionValue>
+                  <ExplorerButton onClick={onClickViewOnBlockExplorer('contract', selectedAsset.contractAddress, selectedAsset.tokenId)}>
+                    <ExplorerIcon />
+                  </ExplorerButton>
+                </ProjectDetailIDRow>
+              </DetailSectionColumn>
+            </DetailSectionRow>
+            <ProjectDetailRow>
+              {/* TODO: Add nft logo when data is available from backend: https://github.com/brave/brave-browser/issues/22627 */}
+              {/* <ProjectDetailImage src={nftMetadata.contractInformation.logo} /> */}
+              <ProjectDetailName>{nftMetadata.contractInformation.name}</ProjectDetailName>
+              {nftMetadata.contractInformation.website && nftMetadata.contractInformation.twitter && nftMetadata.contractInformation.facebook &&
```

Comment:
Just wanted to check here (non-blocking concern) should these be `||`?  If the contract lists a website, but not a twitter handle, don't you still want to display the section?

(I dont really know react so apologies if im reading this wrong)

Reply:
You are right, that is what the code is doing. Design-wise, this was the expectation. I am willing to change it.